stakingModel/contracts/InterestDistribution.sol

```diff
@@ -37,27 +37,28 @@ library InterestDistribution {
       * @param _staker                 Staker's address
       * @param _stake                  Amount of stake
       * @param _donationPer            Percentage will to donate.
-      * @param _interest               Newly accrued interest since last update.
+      * @param _iTokenRate             Interest token rate.
+      * @param _iTokenHoldings         Interest token balance of staking contract. 
       *
     */
     function stakeCalculation(
       InterestData storage _interestData, 
       address _staker,
       uint256 _stake, 
       uint256 _donationPer,
-      uint256 _interest
+      uint256 _iTokenRate,
+      uint256 _iTokenHoldings
       ) 
     internal 
     {
       Staker storage _stakerData = _interestData.stakers[_staker];
-      // Should not update avgYieldRatePerToken for 1st stake as his avg rate should be 0. 
+      // Should not update avgYieldRatePerToken and globalYieldPerToken for 1st stake as his avg rate should be 0. 
       if (_interestData.globalTotalStaked > 0 && _stakerData.stakedToken > 0) {
```

Comment:
why staker.stakedtoken>0?

Reply:
Because, in formula entire thing is divided by TotalStaked, hence it can not be 0. I have confirmed the same with you in slack.
![image](https://user-images.githubusercontent.com/58321840/88261300-a371ad00-cce3-11ea-8a80-1db98715a8b0.png)

Reply:
Please, let me know if there is some miscommunication.